bin/node-template/runtime/build.rs

```diff
@@ -19,7 +19,7 @@ use wasm_builder_runner::{build_current_project_with_rustflags, WasmBuilderSourc
 fn main() {
 	build_current_project_with_rustflags(
 		"wasm_binary.rs",
-		WasmBuilderSource::Crates("1.0.8"),
+		WasmBuilderSource::Crates("1.0.9"),
```

Comment:
Do we also need to bump this on other substrate node template based projects?

Reply:
When you want to always have release" builds, yes.